db/index.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const db = mongoose.connect('mongodb://localhost/tentHop');
+const seed = require('./mockData.js')
+let siteSchema = new mongoose.Schema({
+    siteId: Number,
+    price: Number,
+    weekdayDisc: Number,
+    availability: Array,
+    minStay: {type: Number, default: 2},
+    maxGuests: {type: Number, default: 6},
+  })
+
+  let Site = mongoose.model('Site', siteSchema);
+
+  Site.create(seed.data);
```

Comment:
Do you plan on having it seed data every time the app initialized as opposed to creating a seeding script that the dev would run once?

Reply:
I haven't decided because I am wanting data that shows availability starting from the moment of initialization. That way any dates not included in the database can default to not available including anything prior to the current day. That said, for our purposes it might not matter.

Reply:
ah, but considering it again I DO want to separate the seeding from the index. No need to have it running every time I want to connect to the database! Wanted to get it running, but will definitely write a separate file for it.

Reply:
Last thought on this (for now) after talking to help desk, I'm simplifying to just cover a set of 3 months so that I can focus on front end. Will modify if I end up needing it to work differently! Thanks for pushing more thought on this!